src/stylesheets/components/_mobile-navigation.scss

```diff
@@ -32,6 +32,7 @@
       display: block;
       padding: 16px $govuk-spacing-scale-4 17px $govuk-spacing-scale-4;
       text-decoration: none;
+      @include govuk-typography-weight-bold; // Override .govuk-link weight
```

Comment:
Can we put includes at the top of the rulesets?

Reply:
Updated!